src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.scss

```diff
@@ -39,4 +47,22 @@
     align-items: center;
     justify-content: center;
   }
+
+  .dataSourceAggregatedViewOuiFlexItem {
+    color: grey;
```

Comment:
should we use the oui color guide?

Reply:
I refer to the guides here: https://oui.opensearch.org/1.6/#/guidelines/colors and updated to `5B6875` which is `grey` from the guideline, also confirmed from the local